packages/api/tests/lambdas/sf-event-sqs-to-db-records/test-index.js

```diff
@@ -275,7 +280,8 @@ test.after.always(async (t) => {
     testDbName: t.context.testDbName,
   });
   await cleanupTestIndex(t.context);
-  await sns().deleteTopic({ TopicArn }).promise();
+  await sns().deleteTopic({ TopicArn: ExecutionsTopicArn }).promise();
+  await sns().deleteTopic({ TopicArn: PdrsTopicArn }).promise();
```

Comment:
For my understanding: These are just to make the logic work, and actual testing is in more targeted files, correct?

Reply:
Hm, what do you mean? The actual testing is in more targeted files but I have these two lines to delete the topics since I won't need them at the end of the test

Reply:
'These' referring to having the topics here in general to make the logic run, not the deletions in particular.

Reply:
Ah gotcha, yeah an error is thrown if the topics don't actually exist